setup.py

```diff
@@ -36,14 +36,15 @@ def _get_long_description():
             'License :: OSI Approved :: MIT License',
             'Natural Language :: English',
             'Programming Language :: Python',
-            'Programming Language :: Python :: 3.7',
             'Programming Language :: Python :: 3.8',
             'Programming Language :: Python :: 3.9',
             'Programming Language :: Python :: 3.10',
+            'Programming Language :: Python :: 3.11',
+            'Programming Language :: Python :: 3.12',
         ],
         install_requires=[
-            'Django>=2.2',
+            'Django>=3.2',
```

Comment:
Did you intend to say 4.2 here? Why 3.2?

https://endoflife.date/django

Reply:
@ulgens any news?

Reply:
Updated now.